distributed/comm/registry.py

```diff
@@ -60,8 +62,25 @@ def get_local_address_for(self, loc):
 def get_backend(scheme):
     """
     Get the Backend instance for the given *scheme*.
+
+    New backends can be registered using setuptools entry_points, e.g.:
+    setup(..., entry_points={
+            'distributed.comm.backends': [
+                'udp=dask_udp.backend:UDPBackend',
+            ]
+        },
+    )
     """
-    backend = backends.get(scheme)
+
+    backend = backends.get(scheme) or next(
+        iter(
+            backend_class_ep.load()()
+            for backend_class_ep in pkg_resources.iter_entry_points(
+                "distributed.comm.backends", scheme
+            )
+        ),
+        None,
```

Comment:
This is picking the "first" plugin that is registered with "scheme", in case one exists, otherwise it returns `None`.